src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -277,6 +278,7 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     ov::pass::Manager decompression_handling_manager;
     decompression_handling_manager.set_per_pass_validation(false);
     CPU_REGISTER_PASS_COMMON(decompression_handling_manager, ov::pass::InitNodeInfo);
+    CPU_REGISTER_PASS_COMMON(decompression_handling_manager, ConvertToGatherCompression);
```

Comment:
Just for the record, usual CPU-specific ops are converted in `Transformations::CpuSpecificOpSet`, but `ConvertToGatherCompression` is special because part of the pattern would be matched by `Snippets` or `LPT` related transforms if we do not convert to `GatherCompression` earlier.